tests/mtouch/ErrorTest.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+
+using NUnit.Framework;
+
+using Xamarin.Bundler;
+
+#if MONOTOUCH
+using ProductException = Xamarin.Bundler.MonoTouchException;
+#else
+using ProductException = Xamarin.Bundler.MonoMacException;
+#endif
+
+namespace Xamarin.Test.Bundler {
+
+	[TestFixture]
+	public class ErrorTest {
+
+		[Test]
+		public void Ctor2 ()
+		{
+			var e = new ProductException (0, "valid");
+			Assert.That (e.ToString (), Is.EqualTo ($"warning {ProductException.Prefix}0000: valid"), "ToString-1");
```

Comment:
Why use `Assert.That (..., Is.EqualTo...)` instead of `Assert.AreEqual ()`?

Reply:
it's the newer (now quite old) fluent syntax
it also was the only one part aof NUnitLite when we started monotouch
and the older API, like `AreEqual`, came back in later versions (and eventually NUnitLite was merged back w/Nunit 3)